autoprotocol/types/protocol.py

```diff
@@ -193,26 +200,26 @@ class GelPurifyExtract:
 @dataclass
 class FlowCytometryLaser:
     channels: List[FlowCytometryChannel]
-    excitation: Union[str, Unit] = field(default=None)
-    power: Union[str, Unit] = field(default=None)
+    excitation: WAVELENGTH = field(default=None)
+    power: POWER = field(default=None)
```

Comment:
do we have to worry about backwards compat where strings were passed to these classes in the past?